aws/sdk-codegen/src/main/kotlin/software/amazon/smithy/rustsdk/AwsEndpointDecorator.kt

```diff
@@ -45,13 +47,15 @@ class AwsEndpointDecorator : RustCodegenDecorator {
     }
 
     override fun configCustomizations(
+        rustSettings: RustSettings,
```

Comment:
also, is rustSettings actually used in this PR? I don't disagree with making it available from customizations, but I also don't see any usages